### DISCUSSION TITLE
RaspAP dependencies

### DISCUSSION THREAD

**Discussion by RogueWarrior65** [RaspAP/raspap-webgui#836] [Q&A]
I'm looking to port this to a Yocto-based system and I'd like to know what the dependencies are.
I've ported a much earlier version but knowing what system utilities and versions of things are required in an arbitrary Linux distro would be a useful addition to the docs.

**Comment by billz** [comment]
https://github.com/RaspAP/raspap-webgui/blob/0c056b3833036030982f64562b1a92815895203a/installers/common.sh#L139

You can also find this by searching the official docs https://docs.raspap.com/

**Reply by RogueWarrior65** [reply]
Thanks but not what I was looking for.  Also, Yocto doesn't necessarily include apt-get.
I'm more interested in the various command-line apps used to do things.  In later versions of Yocto, a series of wireless networking tools were replaced by iw.  I can manually install wireless-tools but what other apps are being used and which versions?

**Reply by billz** [reply]
grep /includes for `system` and `exec` calls.
